pkg/reconciler/autoscaling/kpa/kpa.go

```diff
@@ -236,7 +239,9 @@ func reportMetrics(pa *pav1alpha1.PodAutoscaler, pc podCounts) error {
 //    | -1   | >= min | >0    | active     | active     |
 func computeActiveCondition(ctx context.Context, pa *pav1alpha1.PodAutoscaler, pc podCounts) {
 	minReady := activeThreshold(ctx, pa)
-	if pc.ready >= minReady {
+	if pc.ready >= minReady ||
+		// For the case of upgrading an existing service (created pre-0.17)
```

Comment:
Should we remove this after 0.18 cuts?

Reply:
Yep. We don't allow users to skip versions right?

Reply:
We want probably +/-2 just for sake of being good citizens, but in general +/-1 is what we do.

Reply:
I see. Should we say 0.19 then?

Reply:
SG.